codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/rustlang/RustWriter.kt

```diff
@@ -202,9 +208,6 @@ fun <T : CodeWriter> T.docs(text: String, vararg args: Any, newlinePrefix: Strin
     setNewlinePrefix(newlinePrefix)
     // TODO: Smithy updates should remove the need for a number of these changes
     val cleaned = text.lines()
-        // We need to filter out blank lines—an empty line causes the markdown parser to interpret the subsequent
-        // docs as a code block because they are indented.
-        .filter { it.isNotBlank() }
```

Comment:
did you verify this was safe to remove?

Reply:
we should review the output of cargoDocs, I remember a lot of docs were broken by this